.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish multi platform Docker image
+
+on:
+  release:
```

Comment:
I wonder if we should publish the docker image on commit rather than just release. What do you think?

Reply:
I would say the release is good. Because you can match exact versions between git hub and docker hub. Otherwise it will be difficult to use auto versioning. 
You could certainly create an action that automatically creates a release when you commit to main

Reply:
cool, then let's go with releases!